src/ModuleConfig.php

```diff
@@ -22,7 +22,6 @@ public function __invoke()
                     'graphql.processor' => ProcessorFactory::class,
                     'graphql.schema'    => Schema\SchemaFactory::class,
                     GraphQLMiddleware::class => GraphQLMiddlewareFactory::class,
-                    GraphQLMiddlewareDelegatorFactory::class => GraphQLMiddlewareDelegatorFactory::class
```

Comment:
Actually this is the middleware pipeline that is executed when a request hit the server, i have to activate the `raiseThrowable` to handle the error with the json error handler (the raiseThrowables is needed until expressive 1.1 is released).

The factory is needed and not useless, because it's used to set the ErrorHandler.

About the error: wich version are you using of zend-servicemanager?